### PR TITLE
Details

### PR DESCRIPTION
**Pull request by iOiurson**

### CODE REVIEW

File: config/prod.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (DavidBruant) [inline config/prod.json]: Don't version

File: config/prod.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+	"port": 9000,
+	"https": false
```
Comment (DavidBruant) [inline config/prod.json]: should be true

File: example/client.js [hunk 3]
```diff
@@ -8,6 +8,7 @@ var SkyViewControls = require('city-blocks/controls/SkyView_Basic.js');
 var FirstPersonControls = require('city-blocks/controls/FirstPerson_Basic.js');
 
 var cityAPIOrigin = "http://localhost:9000";
+// var cityAPIOrigin = "https://city-api.ants.builders";
```
Comment (DavidBruant) [inline example/client.js]: remove comment

File: front/index.js [hunk 4]
```diff
@@ -128,7 +120,8 @@ module.exports = function(container, buildingServerOrigin, options){
                     server.getCityObject(id);
                 });
             },
-            camera : camera
+            camera : camera,
+            scene: scene
```
Comment (DavidBruant) [inline front/index.js]: Only expose a function to do raycasting

File: server/index.js [hunk 5]
```diff
@@ -118,7 +106,7 @@ metadataP.then(function(metadataString){
 
 server.listen(PORT, function () {
     console.log('Server running on', [
-        config.https ? 'https://' : 'http://',
+        'http://localhost:',
```
Comment (DavidBruant) [inline server/index.js]: add `config.https ? 'https://' : 'http://'`, back

**Comment by DavidBruant** [comment]
You can merge when comments are addressed.

**Comment by iOiurson** [comment]
Raycasting issue will be fixed in further PR. Merging.

**Pull request merged**